Makefile

```diff
@@ -92,21 +104,31 @@ azuredisk-windows:
 	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/azurediskplugin.exe ./pkg/azurediskplugin
 
-.PHONY: container
-container: azuredisk
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/azurediskplugin/Dockerfile .
-
 .PHONY: azuredisk-container
-azuredisk-container: azuredisk
+azuredisk-container:
+# ifdef CI
```

Comment:
why comment here?
The two dependencies have been merged: https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/340, https://github.com/kubernetes/test-infra/pull/17020

Reply:
I was making sure that the CI for Linux cluster pass. I will uncomment it after the new test image reflects kubernetes/test-infra#17020.